### ISSUE TITLE
Compiling Codelite on Mac OS X Sierra

### ISSUE THREAD

**Issue by Kanimies** [eranif/codelite#1732]
Following the instructions at http://codelite.org/Developers/BuildingCodeLiteOnAMac compilation of Codelite (make) fails with error `ld: library not found for -lcrypto` - which library should be used or should the build be successfull on OS X 10.12.5 or 10.12.6?
When using OpenSSL libraries from Homebrew, the build is successful but the program crashes after or during every SFTP operation and I suspect these might be related issues.

**Comment by eranif** [comment]
CodeLite is not using OpenSSL, it uses `libssh` ( https://www.libssh.org/ )
Under the SDK folder `codelite/sdk/libssh/lib/osx` there is a static lib of `libssh.a` built for OSX.
If the builds fails for you, you will need to build `libssh.a` for OSX by yourself and copy the resulted (`libssh.a`) binary into that folder

**Comment by Kanimies** [comment]
Not familiar with building software on Mac (but necessitated to do so as the weekly builds still aren't being updated) I'm not certain if the build is now failing because of the libssh.a that comes with CodeLite or simply because the build process cannot find it from the path it is at?
Also building the libssh library says
```
-- Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the system variable OPENSSL_ROOT_DIR (missing: OPENSSL_INCLUDE_DIR) 
-- Could NOT find GCrypt, try to set the path to GCrypt root folder in the system variable GCRYPT_ROOT_DIR (missing: GCRYPT_INCLUDE_DIR GCRYPT_LIBRARIES) 
CMake Error at CMakeLists.txt:57 (message):
  Could not find OpenSSL or GCrypt
```

which does lead me to believe that CodeLite is in fact using OpenSSL, it's just included within the libssh package?

**Comment by eranif** [comment]
This is the relevant line in https://github.com/eranif/codelite/blob/master/CMakeLists.txt#L225

**Comment by eranif** [comment]
I have released CodeLite 10.0.7 please download it instead
